src/components/dashboard/CustomizeNavbar.js

```diff
@@ -0,0 +1,53 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import React, { useEffect, useState } from 'react';
+
+function CustomizeNavbar({ formFields, setFormFields }) {
+  const handleChangeInput = (e, index) => {
+    const values = [...formFields];
+    const property = e.target.name;
+    values[index][property] = e.target.value;
+    setFormFields(values);
+  };
+
+  return (
+    <div>
+      {formFields.map((formField, index) => (
```

Comment:
You can use the `formField` to get the values instead of using the `index` on the `formFields` to get the values